### DISCUSSION TITLE
page loading

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#45] [Q&A] [answered]
How can page loading time be reduced?

**Comment by Bunny77K9** [comment, marked as answer]
There are many factors that affect the page loading time of a website. However, some methods can be implemented to reduce it drastically. They are given below:

    Reduction in the image size
    Removal of unnecessary widgets
    HTTP compression
    Reduction in lookups
    Minimal redirection and caching
